lib/helpers.js

```diff
@@ -21,5 +23,36 @@ var getLogger = function getLogger(optLogLevel, optTransport) {
   return bind(logger.log, logger);
 };
 
+// TODO we can do this better
+var versions = [
+  [module.exports.repository.replace('/', ':'), module.exports.version],
+  [os.platform(), os.release()],
+  ['node', process.version.replace('v', '')]
+];
+
+var getVersionString = function getVersionString() {
+  var str = '';
+  var i;
+
+  for (i = 0; i < versions.length; ++i) {
+    if (Array.isArray(versions[i])) {
```

Comment:
oh i see, its because the `appendToVersionString()` function adds user-generated.